styles/sass/base/_page.scss

```diff
@@ -0,0 +1,22 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    scroll-behavior: smooth;
+}
+
+*:focus {
+    outline: 2px solid #0C56BC;
+    border-radius: 2px;
+
+}
+
+body {
+    font-family: 'Open Sans',Arial, Helvetica, sans-serif;
```

Comment:
missing space after comma `,`

Reply:
*.scss files should be linted on commit (like the *.astro and *.json) using `prettier --write`

Reply:
I do not understant second comment

Reply:
It should be added into lint-staged 

I've added it in #4  
👇
https://github.com/karczujr/Zasun.eu---Astro/pull/4/files#diff-9fc9dcfe2d0d7daae39f9f1548557d89779a652bedf0c975bbcb3efb8a3d6e25R3